dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -185,4 +185,7 @@ setHtmlContentTypeOnError.name=Disable text/html content type on error responses
 setHtmlContentTypeOnError.desc=When this property is false, the webcontainer will not set the response's content type header during the error handling process. An application is responsible to set the response's content type. The default value (true) sets the content type to "text/html".
 
 excludeAllHandledTypesClasses.name=Disable returning handlesTypes parameters
```

Comment:
Disable returning handlesTypes parameters
--->
Disable the return of handlesTypes parameters